threem.py

```diff
@@ -431,7 +431,12 @@ def __init__(self, _db, input_identifier_types=None,
         )
 
     def process_item(self, identifier):
-        metadata = self.api.bibliographic_lookup(identifier)
+        # We don't accept a representation from the cache because
+        # either this is being run for the first time (in which case
+        # there is nothing in the cache) or it's being run to correct
+        # for an earlier failure (in which case the representation
+        # in the cache might be wrong).
+        metadata = self.api.bibliographic_lookup(identifier, max_age=0)
```

Comment:
Does this also need to be updated in the Overdrive & Axis 360 BibliographicCoverageProviders?